modules/combined/tests/elastic_thermal_patch/tests

```diff
@@ -1,33 +1,71 @@
 [Tests]
-  [./test]
+  [./elastic_thermal_patch]
     type = 'Exodiff'
     input = 'elastic_thermal_patch_test.i'
-    exodiff = 'out.e'
+    exodiff = 'elastic_thermal_patch_out.e'
     abs_zero = 1e-08
     use_old_floor = True
+    prereq = 'elastic_thermal_patch_sm'
   [../]
 
-  [./test_jac_rz_smp]
+  [./elastic_thermal_jacobian_rz_smp]
     type = 'Exodiff'
     input = 'elastic_thermal_jacobian_rz_smp_test.i'
-    exodiff = 'out_jac_rz_smp.e'
+    exodiff = 'elastic_thermal_jacobian_rz_smp_out.e'
     custom_cmp = 'elastic_thermal_patch.cmp'
     use_old_floor = True
     max_parallel = 1
+    prereq = 'elastic_thermal_jacobian_rz_smp_sm'
```

Comment:
We're already using prereqs to have the sm and tm versions of these tests compare against the same gold file.  Could we extend that and do the same for some of the other variants of the test (like the smp one) so that we have fewer gold files?  I'm not sure whether they are otherwise identical, but that would be nice to cut down on gold files if possible.